config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: route-monitor-operator
+  newName: controller
```

Comment:
this has hit us multiple times.. maybe we should have some validation on this file / where we change this field?

https://github.com/openshift/route-monitor-operator/blob/229da76cc133b1226eb69ecb1a7aaafa70563c92/Makefile#L64